clients/drcachesim/tools/opcode_mix.cpp

```diff
@@ -225,10 +227,12 @@ opcode_mix_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             return false;
         }
         opcode = instr_get_opcode(&instr);
-        shard->worker->opcode_cache[trace_pc] = opcode;
+        category = instr_get_category(&instr);
+        shard->worker->opcode_category_cache[trace_pc] = opcode_category_data_t(opcode, category);
```

Comment:
This line looks too long: shouldn't clang-format have split it at 90?

Reply:
Yes, made me realize I had broken my vim config and clang-format wasn't running after saving. Fixed, thanks!